gae/js/app/views/sitecaptain.js

```diff
@@ -68,11 +68,15 @@ define(
                 m.destroy();
                 this.render();
             },
-            addCaptain: function() {
-                console.log('add captain ', this);
+            addCaptain: function(e) {
+                e.preventDefault();
                 var self = this;
+                var choices = this.form.fields.models.find(function(model){return model.get('name') == 'captain';});
+                var captain_id = this.model.get('captain');
+                var captain  = _.findWhere(choices.get('options'), {value: captain_id});
```

Comment:
You can see the changes here: https://issue357-dot-rebuildingtogethercaptain-hrd.appspot.com/room/staff_home